steal.lic

```diff
@@ -161,7 +160,7 @@ class Steal
     end
 
     {
-      'new' => true, 'room' => Room.current.id, 'province' => 'zoluren',
+      'new' => true, 'room' => Room.current.id, 'steal_town' => @steal_town[0],#steal_town replaced province
```

Comment:
I don't see why someone would want to steal from another town but they could.  I could see Leth being another town a thief would want to go to because of the tunnel but an non-thief wouldn't because of the ferry.   When creating a new item the first town on the list is the default value.